src/app/main.cpp

```diff
@@ -1,11 +1,11 @@
 #include <QApplication>
 
-#include "schrodinger/sketcher/sketcher_widget.h"
+#include "schrodinger/sketcher/example_widget.h"
 
 int main(int argc, char** argv)
 {
     QApplication application(argc, argv);
-    schrodinger::sketcher::SketcherWidget sketcher_widget;
-    sketcher_widget.show();
+    schrodinger::sketcher::ExampleWidget widget;
+    widget.show();
```

Comment:
Just to make sure that I'm reading things correctly, there's currently no actual Sketcher code in this repo, right?  It just builds the dependencies plus a dummy Hello-World-ish app, right?

Reply:
yup! I'm gonna move all the actual code over as a separate (probably set of) commit(s).